### DISCUSSION TITLE
Every event marquee effect?

### DISCUSSION THREAD

**Discussion by seewong** [MMRIZE/MMM-CalendarExt3#116] [Q&A]
With the latest update, all my event names have the marquee effect regardless of line length. Anyone else having this issue?

**Comment by eouia** [comment]
Hmmm it should be activated on only long title.
Can you give me your config.js and custom.css?

**Reply by seewong** [reply]
It was doing it right after I restarted the calendar. I left it overnight and it stopped so it should be fine, sorry!
